pythran/tests/__init__.py

```diff
@@ -192,7 +192,10 @@ def run_pythran(self, modname, module_path, runas, prelude=None,
         """
         # Caller may requires some cleaning
         prelude and prelude()
-        pymod = load_dynamic(modname, module_path)
+        loader = importlib.machinery.ExtensionFileLoader(modname, module_path)
+        spec = importlib.machinery.ModuleSpec(name=modname, loader=loader,
+                                              origin=module_path)
+        pymod = importlib._bootstrap._load(spec)
```

Comment:
I guess this could be a function if it is used elsewhere

Reply:
It's also used in `pythran/magic.py`, if you're brave enough :-)

Reply:
done